app/scripts/OverlayTrack.js

```diff
@@ -11,6 +11,34 @@ class OverlayTrack extends PixiTrack {
     this.drawnRects = {};
   }
 
+  drawHorizontalOverlay(graphics, position, extent) {
```

Comment:
```suggestion
  drawHorizontalOverlay(graphics, position, extent = [0, 1]) {
```

Reply:
Resolved by checking if extent exists and is long enough and doing nothing if either is bad.